exit/index.js

```diff
@@ -0,0 +1,45 @@
+require('dotenv').config();
+const Web3 = require('web3');
+const MaticPOSClient = require('@maticnetwork/maticjs').MaticPOSClient;
+const Tx = require('ethereumjs-tx').Transaction;
+
+
+const parentProvider = 'https://mainnet.infura.io/v3/' + process.env.INFURA_API_KEY;
+const maticProvider = 'https://rpc-mainnet.maticvigil.com/';
+const senderPrivateKey = process.env.PRIVATE_KEY;
+const rootChainManagerAddress = 0xBbD7cBFA79faee899Eaf900F13C9065bF03B1A74;
```

Comment:
This is still the goerli address I believe.  You can pull the mainnet address from https://github.com/maticnetwork/static/blob/master/network/mainnet/v1/index.json

Reply:
Fixed